### DISCUSSION TITLE
Feature: multi-line editor

### DISCUSSION THREAD

**Discussion by wandri** [iddan/react-spreadsheet#355] [Ideas]
**Usecase**: Edit of long text with ease.

**Current problem**: The editor is an `<input>` so we can only have a one-line editor. Result, we can just see a part of the edited text.

**Suggested solution**: For long text, it could be interesting to have a multi-line editor in order to be able to see all the text.

Example with google sheet.
![image](https://github.com/iddan/react-spreadsheet/assets/15016833/921a2e5a-0b8f-42fd-a682-e2b9171445a4)

It could be achieve with `<text-area>` or with a `contenteditable="true"`

**Comment by iddan** [comment]
Have you tried customizing the DataEditor component for that?

**Reply by rau** [reply]
@iddan would you be able to provide any examples of customized DataEditors? it would make it super helpful to understand how to actually manipulate the input to be a textarea instead

**Reply by iddan** [reply]
Please see in the stories: https://github.com/iddan/react-spreadsheet/blob/master/src/stories/Spreadsheet.stories.tsx#L188
